admin/query/src/main/java/org/codice/ditto/replication/admin/query/status/fields/ReplicationStats.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.admin.query.status.fields;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Date;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.codice.ddf.admin.api.Field;
+import org.codice.ddf.admin.common.fields.base.BaseObjectField;
+import org.codice.ddf.admin.common.fields.base.scalar.LongField;
+import org.codice.ddf.admin.common.fields.common.PidField;
+import org.codice.ditto.replication.admin.query.replications.fields.Iso8601Field;
+import org.codice.ditto.replication.admin.query.replications.fields.ReplicationStatus;
+
+public class ReplicationStats extends BaseObjectField {
```

Comment:
✏️ Most of our field classes have a name of *field by convention. I think it helps convey that the class is just a graphQL field at a glance.